python/ray/autoscaler/aws/node_provider.py

```diff
@@ -399,3 +399,8 @@ def _get_cached_node(self, node_id):
     def cleanup(self):
         self.tag_cache_update_event.set()
         self.tag_cache_kill_event.set()
+
+    @staticmethod
+    def bootstrap_config(cluster_config):
+        from ray.autoscaler.aws.config import bootstrap_aws
```

Comment:
You could actually move the import to the top of the file for these guys, since they are already in the package.

Reply:
Done.